db/models.py

```diff
@@ -1 +1,35 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return self.name
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The `bonus` field is defined as a `CharField`, but it might be more appropriate to use a different field type if `bonus` is intended to store numerical values or specific choices. Consider using `IntegerField` or `FloatField` for numerical bonuses, or `ChoiceField` if there are predefined bonus types.